### DISCUSSION TITLE
How important is the data in sqlite?

### DISCUSSION THREAD

**Discussion by kenske** [release-argus/Argus#129] [Q&A] [answered]
I just found out about this project and it looks great. I'd like to deploy it to one of my K8s clusters, but in the changelog I saw that it recently added a dependency on sqlite to maintain state. I'd like to avoid having to setup persistent storage if possible. How critical is the data in sqlite? Does it get recreated if the container is replaced?

**Comment by JosephKav** [comment, marked as answer]
The data in SQLite is just used as a store between restarts, so rerunning without the db will just act as though everything without a `deployed_version` is already on the latest version.

TLDR: it'll run fine without a persistent db

**Reply by kenske** [reply]
I think this will work fine for me, but maybe for others it would be useful for Argus to support an external DB (or maybe redis?) so that the state is more reliable.
